src/app/server/Mdns.cpp

```diff
@@ -122,28 +122,23 @@ CHIP_ERROR AdvertiseOperational()
 /// Set MDNS commissioner advertisement
 CHIP_ERROR AdvertiseCommisioner()
 {
-    return Advertise(false);
+    return Advertise(false, false, false);
 }
 
 /// Set MDNS commissionable node advertisement
-CHIP_ERROR AdvertiseCommissionableNode()
+CHIP_ERROR AdvertiseCommissionableNode(bool commissioningMode, bool additionalCommissioning)
 {
-    return Advertise(true);
+    return Advertise(true, commissioningMode, additionalCommissioning);
 }
 
-/// commissionableNode
-// CHIP_ERROR Advertise(chip::Mdns::CommssionAdvertiseMode mode)
-CHIP_ERROR Advertise(bool commissionableNode)
+/// Set MDNS commissioner advertisement or commissionable node advertisement
+CHIP_ERROR Advertise(bool commissionableNode, bool commissioningMode, bool additionalCommissioning)
```

Comment:
Also, should that `additionalCommissioning` arg be `notInitialCommissioning`, or perhaps reversed and `initialCommissioning`?  It's not 100% clear what "additional" means....

Reply:
changed to bitmap and added comments on flag definitions